nixos/modules/misc/locate.nix

```diff
@@ -192,6 +195,19 @@ in {
       { LOCATE_PATH = cfg.output;
       };
 
+    environment.etc = {
+      # write /etc/updatedb.conf for manual calls to `updatedb`
+      "updatedb.conf" = {
+        text = ''
+          PRUNEFS="${lib.concatStringsSep " " cfg.pruneFS}"
+          PRUNENAMES="${lib.concatStringsSep " " cfg.pruneNames}"
+          PRUNEPATHS="${lib.concatStringsSep " " cfg.prunePaths}"
+          PRUNE_BIND_MOUNTSFR="${lib.boolToString cfg.pruneBindMounts}"
+        '';
+        mode = "0444";
```

Comment:
Why do you need this?

Reply:
@milahu I meant, why do you need to set the `mode`? The default `mode` is a symlink to `/nix/store/`.

Reply:
to mark the file as read-only

Reply:
It will be read only anyway.